test/browser_tests/cucumber-new/features/suites/apps/complaint_data_use.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Submit a complaint and Consumer Complaint Database
```

Comment:
Let's drop this file. More basic link navigation checking that is brittle, hard to keep up-to-date, and would better be served by crawling for broken links.

Reply:
Removed this feature file.